### ISSUE TITLE
Assigning roles to existing users

### ISSUE THREAD

**Issue by pdostal** [consul/consul#3950]
### User Story

As the default `Administrator` I want to assign roles to other users.

### Why is this Needed?

Different people need to have Admin, Moderator, Valuator or Manager roles.

### Description

There is 'Users' section in the 'Administration' menu:
 * I can see myself in the Administrators group.
 * I can see the others 'level₃_user' users.

### Current Behavior
There is no way how to assign roles to users.

### Expected Behavior
I expect to be able to control which user has which role.

![image](https://user-images.githubusercontent.com/1254493/76876957-03a69d00-6873-11ea-90bd-207e6df81bc6.png)
![image](https://user-images.githubusercontent.com/1254493/76877055-1c16b780-6873-11ea-92d2-f4c17d8eeefc.png)

**Comment by PierreMesure** [comment]
Hi @pdostal, I think the answer to your problem can be found in the admin manual. I agree the UX for this is not that intuitive though.

**Comment by decabeza** [comment]
Hi @pdostal  It's not very intuitive but it can be done.

For example, in the search form of `/admin/administrators` you can search for any registered user and if they don't have that role you can add them.

![Screenshot 2020-04-27 at 13 05 50](https://user-images.githubusercontent.com/631897/80365431-de507a00-8887-11ea-8b7a-29e1663d6398.png)

In the same way for the rest of the sections/roles. 😉

**Comment by javierm** [comment]
@pdostal Hope the answers help!

I'm keeping the issue open, though, because the user interface should be more intuitive; right now it's pretty much impossible to figure it out.

**Comment by pdostal** [comment]
Hello guys, thank you for the answer! My problem has been solved.

If you plan to do some changes here, feel free to keep this open 😄

**Comment by Michael1337** [comment]
I agree: You will only find this feature if you actively look for it. The current placeholder reads "Search user by name or email". Changing that to "Search all users [...]" would be helpful. Not sure how that is handled in other languages. (Also, the English text as it is right now should actually be "Search for user [...]", if I am not mistaken.)

**Comment by jfoclpf** [comment]
Thanks @javierm and @decabeza 
btw, how to add to an user the role of Organisation? That method you mentioned does not work

**Comment by taitus** [comment]
Hi @jfoclpf

It is not possible to add a user to the organization role through the administration panel. 
This role works differently from the others as it needs a name for the organization and a name for the responsible person.

A user with an organization role is created from the sign up, through the link: 
![Organization sign up](https://user-images.githubusercontent.com/16189/99652312-f5d79600-2a57-11eb-9b41-de27f18280e4.png)

Another possible solution would be to do it by console.

**Comment by jfoclpf** [comment]
Thank you very much @taitus 
yes, now at see that organizations must register at `/organizations/sign_up`

**Comment by silberzwiebel** [comment]
> @pdostal Hope the answers help! 😄
> 
> I'm keeping the issue open, though, because the user interface should be more intuitive; right now it's pretty much impossible to figure it out.

As I was stumbling over this issue and couldn't find the answer you were writing about, here is what I finally found out (please change this in the UI :) ) for the hopeful benefit of future readers:

If you want to give a user higher rights (e.g., moderator, valuator etc.), you have to 

- click on the corresponding group in the left panel
- search for the user you want to give the rights in the search box on top
- click the "add user" button in the result list

**Comment by javierm** [comment]
@silberzwiebel Thanks! I'm pretty sure there was an answer by decabeza saying what you just mentioned, but I can't find it anymore.
